src/aiod_sdk/calls/calls.py

```diff
@@ -0,0 +1,174 @@
+import aiohttp
+import asyncio
+import requests
+
+import pandas as pd
+from typing import Literal
+from functools import partial
+
+from aiod_sdk.calls.utils import (
+    url_to_get_asset,
+    url_to_get_list,
+    url_to_resource_counts,
+    format_response,
+    wrap_calls,
+)
+
+
+def get_list(
+    *,
+    asset_type: str,
+    offset: int = 0,
+    limit: int = 10,
+    version: str | None = None,
+    data_format: Literal["pandas", "json"] = "pandas",
+) -> pd.DataFrame | list:
+    """
+    Retrieve a list of ASSET_TYPE from the catalogue.
+
+    Parameters (keywords required):
+        offset (int): The offset for pagination (default is 0).
+        limit (int): The maximum number of items to retrieve (default is 10).
+        version (str | None): The version of the endpoint (default is None).
+        data_format (Literal["pandas", "json"]): The desired format for the response (default is "pandas").
+            For "json" formats, the returned type is a json decoded type, i.e. a dict or a list.
+
+    Returns:
+        pd.DataFrame | dict | list: The retrieved metadata in the specified format.
+    """
+    url = url_to_get_list(asset_type, offset, limit, version)
+    res = requests.get(url)
+    resources = format_response(res.json(), data_format)
+    return resources
+
+
+def counts(
+    *, asset_type: str, version: str | None = None, detailed: bool = False
+) -> int:
+    """
+    Retrieve the counts of ASSET_TYPE.
+
+    Parameters (keywords required):
+        version (str | None): The version of the endpoint (default is None).
+        detailed (bool): Whether to retrieve detailed counts (default is False).
+
+    Returns:
+        int: The count of ASSET_TYPE.
```

Comment:
```suggestion
    Retrieve the number of ASSET_TYPE assets in the metadata catalogue.

    Parameters (keywords required):
        version (str | None): The version of the endpoint (default is None).
        detailed (bool): Whether to list counts per platform (default is False).

    Returns:
        int: The number ASSET_TYPE assets in the metadata catalogue.
```

Is the only form of "detailed" output per platform? If so, I would consider renaming this in the API and SDK to a more descriptive name such as `per_platform`.

Reply:
The signature is also wrong: it depends on whether `detailed` is set to `True` or `False`. If `True`, it actually returns a `dict[str, int]`. And this should be documented.

Reply:
> Is the only form of "detailed" output per platform? If so, I would consider renaming this in the API and SDK to a more descriptive name such as `per_platform`.

Yes, as far as I understand it, the "detailed" output is per platform. I'll rename it. Thanks :)